src/components/Camera/AICamera/FrameProcessorCamera.js

```diff
@@ -125,9 +125,12 @@ const FrameProcessorCamera = ( {
         return;
       }
       const timestamp = Date.now();
-      const timeSinceLastFrame = timestamp - lastTimestamp;
-      if ( timeSinceLastFrame < ( 1000 / fps ) ) {
-        return;
+      // If there is no lastTimestamp, i.e. the first time this runs do not compare
+      if ( lastTimestamp ) {
```

Comment:
This is a small bugfix, the first few frames - from init of screen to the first time the check becomes truthy - were not send to the camera. Now it starts immediately.